src/Microsoft.OpenApi/Writers/OpenApiWriterExtensions.cs

```diff
@@ -458,6 +459,17 @@ private static void WriteMapInternal<T>(
 
             if (elements != null)
             {
+                var settings = writer.GetSettings();
+
+                if (settings?.EnableSorting == true)
```

Comment:
probably can be cleaned up if we remove the setting.
I think people should tell us what sorting they want, if not, we simply rely on the collections' ordering (no additional sorting).